tests/2019-06-01/Microsoft.Storage.tests.json

```diff
@@ -267,7 +267,7 @@
         "type": "Microsoft.Storage/storageAccounts/managementPolicies",
         "apiVersion": "2019-06-01",
         "id": "/subscriptions/{subscription-id}/resourceGroups/res7231/providers/Microsoft.Storage/storageAccounts/sto288/managementPolicies/default",
-        "name": "default",
+        "name": ".a/default",
```

Comment:
Why add ".a/", this is not aligned with real request.

Reply:
Discussing in another comment, so resolve this one.